sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java

```diff
@@ -69,6 +69,15 @@ public static void init(@NotNull final SentryAndroidOptions options) {
 
   /** Closes the NDK integration */
   public static void close() {
-    shutdown();
+    try {
+      if (loadLibraryLatch.await(2000, TimeUnit.MILLISECONDS)) {
```

Comment:
I thinks it's fine this way because `.init()` already waits for the latch to finish, so whenever `.close()` is called the latch is already counted down. So even a double init wouldn't cause this to be blocking, unless you call `SentryAndroid.init()` from different threads in parallel 😅 
So the `.await()` will only be blocking in case `SentryNdk.close()` is called before `SentryNdk.init()` - in theory this should never happen.